java/ql/src/Security/CWE/CWE-094/InsecureBeanValidation.ql

```diff
@@ -38,5 +81,8 @@ class BeanValidationConfig extends TaintTracking::Configuration {
 }
 
 from BeanValidationConfig cfg, DataFlow::PathNode source, DataFlow::PathNode sink
-where cfg.hasFlowPath(source, sink)
-select sink, source, sink, "Custom constraint error message contains unsanitized user data"
+where
+  not exists(SetSafeMessageInterpolatorCall ma) and
```

Comment:
As discussed individually: let's change this condition to `forall(SetMessageInterpolatorCall c | | c.isSafe())`.
This will:
- continue to avoid false positives where the application sets only a safe validator (the motivation for this PR)
- potentially avoid false negatives where the application sets multiple interpolators, some of which are safe and some of which are unsafe
- potentially lead to a few false positives where the application sets multiple interpolators, some of which are safe and some of which are unsafe, and we flag a validated object that actually uses a safe validator. I'm willing to accept that for now.